x-pack/plugins/alerting/common/routes/framework/apis/health/schemas/v1.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+const alertingFrameworkHealthSchemaObject = {
+  status: schema.oneOf([schema.literal('ok'), schema.literal('warn'), schema.literal('error')]),
+  timestamp: schema.string(),
+};
+
+export const healthFrameworkResponseBodySchema = schema.object({
+  is_sufficiently_secure: schema.boolean({
+    meta: {
+      description: 'If false, security is enabled but TLS is not.',
+    },
+  }),
+  has_permanent_encryption_key: schema.boolean({
+    meta: {
+      description:
+        'If false, the encrypted saved object plugin does not have a permanent encryption key.',
```

Comment:
nit: What about "If false, the `xpack.encryptedSavedObjects.encryptionKey` config is not set"? or `If false, the encryption key is not set`?

Reply:
I'll use the latter to avoid mentioning implementation details